webdriver/tests/bidi/script/call_function/result.py

```diff
@@ -119,9 +120,16 @@ async def test_primitive_values(bidi_session, top_context, await_promise, expres
         ("new WeakMap()", {"type": "weakmap"}),
         ("new WeakSet()", {"type": "weakset"}),
         ("new Error('SOME_ERROR_TEXT')", {"type": "error"}),
-        # TODO(sadym): add `iterator` test.
-        # TODO(sadym): add `generator` test.
-        # TODO(sadym): add `proxy` test.
+        ("([1, 2][Symbol.iterator]())", {
+            "type": "iterator",
+        }),
+        ("new Proxy({}, {})", {
+            "type": "proxy"
+        }),
+        # generator
+        ("(function*() { yield 'a'; })", {
```

Comment:
should not that have been `(function*() { yield 'a'; })()`? since the generator is an object returned by the generator function?

Reply:
https://github.com/web-platform-tests/wpt/pull/40914